frontend/src/modules/EditZing/Components/EmailModal.tsx

```diff
@@ -23,10 +32,42 @@ export const EmailModal = ({
   setEmailSentError,
   handleEmailTimestamp,
 }: EmailModalProps) => {
-  const [selectedTemplate, setSelectedTemplate] = useState<TemplateName>(
-    '' as TemplateName
+  // check if emailing students or groups
+  const recipientType = selectedStudents.length > 0 ? 'student' : 'group'
+
+  // template editor logic
+
+  const [templates, setTemplates] = useState<EmailTemplate[]>([])
+  const [templateName, setTemplateName] = useState('')
+  const [selectedTemplate, setSelectedTemplate] = useState<EmailTemplate>(
+    templates[0]
```

Comment:
This kind of removes some of the benefits of having the type system if we lie to it by giving it `templates[0]` (supposedly an EmailTemplate) knowing that `templates[0]` is actually undefined when this code runs.